pkg/inframanager/store/storedefinition.go

```diff
@@ -65,10 +65,54 @@ type ServiceCollection struct {
 	ServiceLock *sync.Mutex
 }
 
+type Policy struct {
+	PolicyName string
+	PolicyId   map[string]PolicyID
```

Comment:
what is the string key in this map? For TCP, UDP in ingress and egress directions, we need four PolicyIDs per Policy.